R/risk_mod.R

```diff
@@ -245,8 +248,8 @@ risk_coord_desc <- function(X, y, gamma, beta, weights, lambda0 = 0,
 #' mod3$model_card
 #' @export
 risk_mod <- function(X, y, gamma = NULL, beta = NULL, weights = NULL,
-                     lambda0 = 0, a = -10, b = 10, max_iters = 100, tol= 1e-5,
-                     shuffle = TRUE, seed = NULL) {
+                     n_train_runs = 5, lambda0 = 0, a = -10, b = 10,
+                     max_iters = 100,  tol = 1e-5, shuffle = TRUE, seed = NULL) {
 
```

Comment:
**question**: Do we need to prevent users from setting the seed when `n_train_runs > 1`? Wouldn't setting the seed make the iterations to optimize accuracy redundant?